python/docs/cpp.rst

```diff
@@ -0,0 +1,8 @@
+Welcome to the rmm C++ documentation!
+========================================
```

Comment:
This shows up in the navigation bar (they call these "breadcrumbs").

![image](https://github.com/rapidsai/rmm/assets/3943761/327cda0e-c7ae-4b36-a5e5-05364a47b2a2)

I would prefer to do something like:
```suggestion
rmm C++
=======
```

Reply:
I agree -- never liked the "Welcome to" prefix on practically every Python package's docs.